launcher/container_runner_test.go

```diff
@@ -414,6 +417,71 @@ type idTokenResp struct {
 	Token string `json:"token"`
 }
 
+func TestFetchContainerImageSignatures(t *testing.T) {
+	ctx := namespaces.WithNamespace(context.Background(), "test")
```

Comment:
We should consider using something like https://pkg.go.dev/github.com/docker/docker/testutil/registry#V2 or the mock https://pkg.go.dev/github.com/docker/docker/testutil/registry#Mock instead of actually making a network call in our unit tests